### PR TITLE
Training report cleanup for testing party

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change

- Putting the event status in a read-only field under certain circumstances meant the submit error message was hidden. This shows it.
- Breakpoints for the event card weren't quite reflecting the content, they have been updated
- There was a possibility for a newly created event to throw an error saving if there was no POC ID set
- getSessionById was missing a variable assignment that threw an error
- Created a seeder (forking this branch to work on an e2e test)

## How to test

Confirm the errors listed above have been resolved.

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: src/seeders/20230718123611-tr-report.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+const { SCOPE_IDS } = require('@ttahub/common');
+
+const {
```
Comment (AdamAdHocTeam) [inline src/seeders/20230718123611-tr-report.js:3]: Any need to disable audit with the seeder stuff?

File: src/routes/sessionReports/handlers.js [hunk 2]
```diff
@@ -42,6 +42,7 @@ export const getHandler = async (req, res) => {
 
     if (id) {
       session = await findSessionById(id);
+      sessionEventId = session.eventId;
```
Comment (AdamAdHocTeam) [inline src/routes/sessionReports/handlers.js:45]: Did we just totally miss this?

File: src/seeders/20230718123611-tr-report.js (hunk 1 above)
Comment (thewatermethod) [reply src/seeders/20230718123611-tr-report.js:3]: @AdamAdHocTeam  No - seeders aren't run in production, and don't populate the audit log

**Review by AdamAdHocTeam — APPROVED** [review]
Looks good.

**Pull request merged**